test/e2e/test/context.go

```diff
@@ -81,8 +77,7 @@ func defaultContext() Context {
 
 // Context encapsulates data about a specific test run
 type Context struct {
-	GlobalOperator        ClusterResource   `json:"global_operator"`
-	NamespaceOperator     NamespaceOperator `json:"namespace_operator"`
+	Operator              NamespaceOperator `json:"global_operator"`
```

Comment:
Should we also rename `global_operator` ?

Reply:
Yes 💯 I relied on the refactoring support of my editor and missed that.